New structure for FAQs (#191)
seqeralabs/docs: platform_versioned_docs/version-24.1/troubleshooting-and-faqs/api-and-cli.mdx

`{object} length parameter cannot be greater than 100 (current value={value_sent})`

To remedy this, see the example requests below:

```
curl -X GET "https://$TOWER_SERVER_URL/workflow/$WORKFLOW_ID/tasks?workspaceId=$WORKSPACE_ID&max=100" \

-H "Accept: application/json" \

-H "Authorization: Bearer $TOWER_ACCESS_TOKEN"

curl -X GET "https://$TOWER_SERVER_URL/workflow/$WORKFLOW_ID/tasks?workspaceId=$WORKSPACE_ID&max=100&offset=100" \

-H "Accept: application/json" \

-H "Authorization: Bearer $TOWER_ACCESS_TOKEN"
```

## tw CLI

**Connection errors when creating or viewing AWS Batch compute environments with `tw compute-envs` commands**

tw CLI v0.8 and earlier does not support the `SPOT_PRICE_CAPACITY_OPTIMIZED` [allocation strategy](./compute-envs/aws-batch.mdx#advanced-options) in AWS Batch. Creating or viewing AWS Batch compute environments with this allocation strategy will lead to errors. This issue will be [addressed in CLI v0.9](https://github.com/seqeralabs/tower-cli/issues/332).

**Segfault errors**

> Users of legacy `tw` CLI versions may experience segmentation faults in older operating systems.

Author: ```suggestion
Users of legacy tw CLI versions may experience segmentation faults in older operating systems.
```

Author: I agree - we're not referencing a command in this context.

> Users of legacy `tw` CLI versions may experience segmentation faults in older operating systems.

Author: I agree. When we're referring to a command, then `tw` works. But in most cases we're using tw to refer to "tw CLI" as a service.

> To resolve segfault errors, first upgrade your tw CLI to the latest available version. If errors persist, use our alternative Java [JAR-based solution](https://github.com/seqeralabs/tower-cli/releases/download/v0.8.0/tw.jar).

Author: ```suggestion
To resolve segfault errors, first upgrade your tw CLI to the latest available version. If errors persist, use our alternative Java [JAR-based solution](https://github.com/seqeralabs/tower-cli/releases/download/v0.10.1/tw-jar.jar).
```

**Insecure HTTP errors**

_ERROR: You are trying to connect to an insecure server: `http://hostname:port/api`if you want to force the connection use '--insecure'. NOT RECOMMENDED!_

This error indicates that your Seqera host accepts connections using insecure HTTP instead of HTTPS. If your host cannot be configured to accept HTTPS connections, add the `--insecure` flag **before** your CLI command:

```
$ tw --insecure info
```

:::caution
HTTP must not be used in production environments.
:::

**Resume/relaunch runs with tw CLI**

> Runs can be [relaunched](./launch/launchpad.mdx#relaunch-pipeline-run) with `tw runs relaunch` .

Reviewer: ```suggestion
Runs can be [relaunched](./launch/launchpad.mdx#relaunch-pipeline-run) with `tw runs relaunch`.
```
